ups/obs_rubinGenericCamera.cfg

```diff
@@ -0,0 +1,13 @@
+# -*- python -*-
+
+import lsst.sconsUtils
```

Comment:
I think you can drop this file if you change the `SConstruct` file to use `noCfgFile=True`.